<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionWidget</class>
 <widget class="QWidget" name="OptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rb_Rtu">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Modbus RTU</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_Tcp">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Modbus TCP</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_Rtu">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Modbus RTU</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>数据位</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="port">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="databits">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="stopbits">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="baudrate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>波特率</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>端口</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>停止位</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="parity">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>奇偶校验位</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Tcp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Modbus TCP</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Port</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>IP</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="tcp_port">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="value">
              <number>8080</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ip">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>rb_Rtu</sender>
   <signal>clicked()</signal>
   <receiver>OptionWidget</receiver>
   <slot>onTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_Tcp</sender>
   <signal>clicked()</signal>
   <receiver>OptionWidget</receiver>
   <slot>onTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>OptionWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTypeChanged()</slot>
  <slot>save()</slot>
 </slots>
</ui>
